hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/map/UpdateProcessorSupplier.java

```diff
@@ -104,7 +103,7 @@ private CompletableFuture<Traverser<Integer>> update(List<JetSqlRow> rows, IMap<
             assert row.getFieldCount() == 1;
             keys.add(row.get(0));
         }
-        return map.submitToKeys(keys, updaterSupplier.get(UntrustedExpressionEvalContext.from(evalContext)))
```

Comment:
Why does SQL updating statement with get_ddl work after putting a trusted context here?
A serialized ExecutionPlan is sent to the nodes, which contains both a Subject and an UpdateProcessorSupplier. Deserialization occurs for UpdateProcessorSupplier. After deserialization, the ExecutionPlan.initProcSuppliers method is called, where the trusted context is set. Everything is fine here.
What is the problem - the problem is in the backup process, it is the sterilized UpdatingEntryProcessor that is send in the operation queue,
Which after deserialization remains with UntrustedEvalContext. And when system try to process backup operation it crashes with the error “Unable to employ sensitive functions in untrusted invocations.” This does seem well.

What to do?
1. return a call to this method with UntrustedEvalContext as I suggested at the beginning.
+ Fast and predictable behavior
- get_ddl in update statement will not work
2. Try to find a way to set the context after deserialization of the UpdatingEntryProcessor, for example here:
![image](https://github.com/hazelcast/hazelcast/assets/150694869/0d2a46e4-12f4-4396-a63f-e2411b1878bd)
+ get_ddl in update statement will work
- I am not sure that this will not affect any other processes.